README.md

```diff
@@ -1 +1,60 @@
-# Hello_Microverse
\ No newline at end of file
+Hello_Microverse
+
+_______________________________________________________________________________
+
+
+Additional description about the project and its features.
```

Comment:
- [ ] Your readme is descriptive, but it is not formatted. When someone opens it, they can see only plain text. I strongly recommend you to heck this [link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to learn about formatting in github. You will need it for the next 6 months 😆 
Or you can use [this template](https://github.com/microverseinc/readme-template) and make sure to customise it 😉

Reply:
ReadMe has been updated